src/Controller/OnboardingWizardStatusController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusMolliePlugin\Controller;
```

Comment:
move controller to right namespace, shop? admin? and on the right folder like Onboarding